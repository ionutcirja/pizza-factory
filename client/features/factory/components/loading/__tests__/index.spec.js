import React from 'react';
import { shallow } from 'enzyme';
import { Loading } from '..';

describe('Loading component', () => {
  const propsToRender = {
    message: 'loading...',
    theme: {
      colours: {
        blue: 'blue',
      },
    },
  };
  
  describe('render', () => {
    it('should render a loading message', () => {
      const wrapper = shallow(<Loading {...propsToRender} />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
