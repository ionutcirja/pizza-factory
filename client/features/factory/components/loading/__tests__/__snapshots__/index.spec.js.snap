// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component render should render a loading message 1`] = `
<span>
  loading...
</span>
`;
